api/kubelb.k8c.io/v1alpha1/route_types.go

```diff
@@ -180,7 +178,7 @@ func ConvertServicesToUpstreamServices(services []corev1.Service) []UpstreamServ
 	return upstreamServices
 }
 
-func ConvertReferenceGrantsToUpstreamReferenceGrants(referenceGrants []gwapiv1a2.ReferenceGrant) []UpstreamReferenceGrant {
+func ConvertReferenceGrantsToUpstreamReferenceGrants(referenceGrants []gwapiv1alpha2.ReferenceGrant) []UpstreamReferenceGrant {
```

Comment:
I think we can use `gateway.networking.k8s.io/v1beta1` for ReferenceGrant, wdyt?

Reply:
If it's okay for you, I'll keep this as it is in the current PR. Reference Grants support is a separate feature so it'll be properly implemented in a separate PR.